modules/oe_starter_content_event/config/install/field.field.node.oe_event.oe_summary.yml

```
@@ -0,0 +1,18 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.oe_summary
+    - node.type.oe_event
+id: node.oe_event.oe_summary
+field_name: oe_summary
+entity_type: node
+bundle: oe_event
+label: Introduction
+description: ''
+required: false
+translatable: false
+default_value: {  }
+default_value_callback: ''
+settings: {  }
+field_type: string_long
```

both the ticket and the example in oe_content say text_long
```suggestion
field_type: text_long
```